### ISSUE TITLE
Use SDL_mixer

### ISSUE THREAD

**Issue by jasp00** [DungMunkey/Dark-Ages#3]
Would you consider using SDL_mixer in the music system instead of SFML for consistency?

**Comment by DungMunkey** [comment]
Yes, certainly, and done. Sorry for the nearly 9 year shelving of this project.
